src/lib/auth/authconf.rb

```diff
@@ -755,18 +755,99 @@ def aux_apply
                 service_disable_stop('nscd')
             end
         end
-        
+
+        def IsInstalledVersionNewerOrEqual(installed_rpm_version, test_rpm_version)
+            installed_rpm_version_l = Yast::Builtins.filter(
+                Yast::Builtins.splitstring(installed_rpm_version, "-.+")
+            ) do |one_item|
+                    Yast::Builtins.regexpmatch(one_item, "^[0123456789]+$")
+                end
```

Comment:
Please, use ruby std instead of Yast::Builtins, if possible:

```suggestion
            installed_rpm_version_l = installed_rpm_version.split(/[-.+]/).select { |i| i.match?(/^[0123456789]+$/) }
```